test/extended/builds/build_pruning.go

```diff
@@ -149,4 +152,34 @@ var _ = g.Describe("[builds][pruning] prune builds based on settings in the buil
 
 	})
 
+	g.It("buildconfigs should have a default history limit set when created via the group api", func() {
```

Comment:
(sorry, these tests in particular do not run builds.  i guess i can add just the new ones to conformance)